lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -147,6 +148,12 @@ def ask_for_tables_to_exclude_in_dump
         end || true
       end
 
+      def ask_to_split_up_output
+        if action != :restore && should_split_output?
```

Comment:
I think we should only support this for the `:dump` action on the first pass since restore from split backups won't work and it wasn't really part of the original RFE.

Reply:
I guess what I'm saying is that we shouldn't give the user the option to backup using splits if we can't restore with splits. I'm fine with the option to dump using splits because the intention was never to restore it.

Reply:
> I guess what I'm saying is that we shouldn't give the user the option to backup using splits if we can't restore with splits.

I guess that is fair.

I figured that someone would eventually request the splitting functionality for backups as well anyway, specifically for `s3`, since there limits to file size.

* * *

Please note that I am also partially arguing this point simply because I don't want to delete the appliance console integration tests that I have written that already support running against and validating split based backups...

(deep down... I am really a simple human being really...)

Reply:
Definitely not arguing against creating split backups at some point, but I think we need split restore at that point as well. If we leave this as is, some user will be rather unhappy when we have to tell them that they can't restore a backup that we let them create.

Reply:
Welp, I can do that.  Despite not wanting to comment out/remove some of my personal tests, I think I agree with that line of thinking.

Will try and take care of that code change shortly.